### ISSUE TITLE
[Adaptive Cards - Blog] - There is no 'H1' heading defined for the blogs page.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8551]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS: Version 23H2 (OS Build 25381.1)
URL: https://adaptivecards.io/blog/
Browser: Edge Dev Version 115.0.1901.5 (Official build) dev (64-bit)
Tool: Accessibility Insight For Web

**Repro Steps:**
1. Open the above URL in the edge dev browser.
2. Run the tool and observe the issue.

**Actual Result:**
There is no H1 heading programmatically and visually defined on the blog page. Only H2 and H3 headings are defined.

**Expected Result:**
There should be programmatically and visually 'H1' heading should be defined on the blog screen. 

**User Impact:**
It will impact the low vision user if there is no H1 heading is visually defined. 

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “,

### Screenshots

![Usable - There is no H1 heading is defined for the Blog screen](https://github.com/microsoft/AdaptiveCards/assets/95913748/2781c255-0a30-4041-996f-d316504d1d07)

### Card JSON

```JSON
Na
```
